CHANGELOG/CHANGELOG-1.12.md

```diff
@@ -0,0 +1,73 @@
+- Add new exporter to FlowAggregator to write flows to local file. ([#4855](https://github.com/antrea-io/antrea/pull/4855), [@antoninbas])
+- Support new OS openEuler 22.03 and add it in tested distributions. ([#4957](https://github.com/antrea-io/antrea/pull/4957), [@ceclinux])
+
+### Changed
+
+- Deprecate Antrea Octant Plugin. ([#4825](https://github.com/antrea-io/antrea/pull/4825), [@antoninbas])
+- Update Open vSwitch version to 2.17.6. ([#4959](https://github.com/antrea-io/antrea/pull/4959), [@tnqn])
+- Update Windows OVS version to 2.16.7. ([#4705](https://github.com/antrea-io/antrea/pull/4705), [@XinShuYang])
+- Upgrade K8s libraries to v0.26.4. ([#4935](https://github.com/antrea-io/antrea/pull/4935), [@heanlan])
+- Bump up whereabouts to v0.6.1. ([#4988](https://github.com/antrea-io/antrea/pull/4988), [@hjiajing])
+- Update Egress API to support multiple Egress IPs and pools. ([#4603](https://github.com/antrea-io/antrea/pull/4603), [@tnqn])
+- Check the existence of AntreaAgentInfo CRD before operating on it for worker Node or ExternalNode. ([#4762](https://github.com/antrea-io/antrea/pull/4762), [@wenyingd])
+- Stop serving v1alpha2 version of the ClusterGroup CRD. ([#4812](https://github.com/antrea-io/antrea/pull/4812), [@antoninbas])
+- Change to cache flow modification messages in Antrea Agent to improve Agent memory usage. ([#4495](https://github.com/antrea-io/antrea/pull/4495), [@wenyingd])
+- Replace PacketIn/Controller with PacketIn2/Controller2 to improve packetin handler. ([#4768](https://github.com/antrea-io/antrea/pull/4768), [@GraysonWu])
+- Change to look up Pods by name instead of IP address to fetch labels in Flow Aggregator, to avoid obtaining incorrect Pods when Pods turn over quickly. ([#4942](https://github.com/antrea-io/antrea/pull/4942), [@dreamtalen])
+- Do not export Services of ExternalName type which is unsupported in upstream Multi-cluster Service. ([#4814](https://github.com/antrea-io/antrea/pull/4814), [@luolanzone])
+- Update Multi-cluster user guide to provide more details for Gateway enablement. ([#4889](https://github.com/antrea-io/antrea/pull/4889), [@luolanzone])
+- Update documentation for recent MetalLB versions. ([#4803](https://github.com/antrea-io/antrea/pull/4803), [@antoninbas])
+- Add support for short-circuiting in AntreaProxy to ensure that the traffic from Pod/Node clients to
+external addresses behaves the same way as the traffic from external clients to external addresses. ([#4815](https://github.com/antrea-io/antrea/pull/4815), [@hongliangl])
+
+### Fixed
+
+- Use LOCAL instead of CONTROLLER as the in_port of packet-out message to fix Windows agent crash issue. ([#4992](https://github.com/antrea-io/antrea/pull/4992), [@tnqn])
+- Fix the Antrea Agent crash issue on Windows by running modules that rely on Services after AntreaProxy is ready. ([#4946](https://github.com/antrea-io/antrea/pull/4946), [@tnqn])
```

Comment:
To be consistent with the previous one, maybe change to "Run agent modules that rely on Services access after AntreaProxy is ready to fix a Windows agent crash issue"